en/docs/install-and-setup/setup/security/configuring-transport-level-security.md

```diff
@@ -82,7 +83,42 @@ By default, all WSO2 products pass "WSO2 Carbon Server" as the server value in H
     server="WSO2 Carbon Server"
     ```
 
-!!! info
-    See the [Security Guidelines for Production Deployment](https://docs.wso2.com/display/ADMIN44x/Security+Guidelines+for+Production+Deployment) for the full list of security-related recommendations for WSO2 products.
+## Disabling HTTP Transports
+
+!!! Note
+    It is recommended to disable the HTTP transport in an API Manager production setup. Using the `Bearer` token over HTTP is a violation of the OAuth specification and can lead to security vulnerabilities.
+
+API Manager has two HTTP transports.
+
+-   Passthru (API Traffic) Transport
+-   Servlet (UI Traffic and Admin service access) Transport
+
+See the instructions given below to disable these transports.
+
+### Disabling Passthrough Transport
+
+Add the following configuration in the `deployment.toml` file (stored in the `<API-M_HOME>/repository/conf` folder).
+
+```toml
+[transport.passthru_http.listener]
+enable = false
+```
+
+### Disabling Servlet Transport
```

Comment:
@tharindu1st I was unable to find the toml configs for this.
According to the config catalog, the the toml configs for the servlet transport is [1]. However, there is no parameter for disabling. Is this disabled by default?

[1] https://apim.docs.wso2.com/en/latest/reference/config-catalog/#http-servlet-transport

Reply:
This was needs to address from WUM update.

Reply:
https://github.com/wso2/product-apim/issues/11177

Reply:
ack. Will leave the doc as it is until the bug is fixed.